.github/workflows/_docs.yml

```diff
@@ -53,6 +53,31 @@ on:
         description: Permissions for pushing to the docs site.
 
 jobs:
+  lint-urls:
+    if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+    name: lint-urls
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
+        with:
+          submodules: false
+          fetch-depth: -1
+      - name: Run URL linter
+        run: bash ./scripts/lint_urls.sh
```

Comment:
Umm, we are trying to move away from bash if possible.  Do you think these scripts could be rewritten in Python or would it be too much throw away work?

Reply:
They likely could (and call shell tools from a Pythol script lol), but that'd be a separate change.